jdaviz/app.py

```diff
@@ -2286,17 +2286,24 @@ def _on_new_viewer(self, msg, vid=None, name=None, add_layers_to_viewer=False):
             msg.cls, data=msg.data, show=False)
         viewer.figure_widget.layout.height = '100%'
 
+        linked_by_wcs = self._link_type.lower() == 'wcs'
```

Comment:
Why would `app._link_type` have mixed case scenario here that requires `lower()`?

Reply:
I had used `lower` mostly because I wasn't sure if I should use upper or lower case in my equality check, since we have upper and lower case WCS's throughout the package. I think it's troublesome to have case-sensitive checks anyway, so I lower them.

Reply:
The hidden attribute should always be lowercase. Only the plugin traitlet should always be upper case because it is set up from the UI dropdown, right?

Reply:
```suggestion
        linked_by_wcs = self._link_type == 'wcs'
```
> Only the plugin traitlet should always be upper case because it is set up from the UI dropdown, right?

If you say so?